### PR TITLE
chore: better types for funcs

### PR DESCRIPTION
**Pull request by kumaraditya303**

### CODE REVIEW

**Comment by kumaraditya303** [comment]
The failing tests are not related and were broken since 06a62bb19e90b26981630f836b4d0a5b6c0a7252

File: playwright/_impl/_transport.py [hunk 1]
```diff
@@ -44,7 +45,7 @@ def _get_stderr_fileno() -> Optional[int]:
 class Transport(ABC):
     def __init__(self, loop: asyncio.AbstractEventLoop) -> None:
         self._loop = loop
-        self.on_message = lambda _: None
+        self.on_message: Callable[[ParsedMessagePayload], None] = lambda _: None
```
Comment (mxschmitt) [inline playwright/_impl/_transport.py:48]: please also change the return type in `deserialize_message` to `ParsedMessagePayload`.

**Review by mxschmitt — APPROVED** [review]
LGTM, just a small nit

File: playwright/_impl/_transport.py (hunk 1 above)
Comment (kumaraditya303) [reply playwright/_impl/_transport.py:48]: Done

**Review by mxschmitt — APPROVED** [review]

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**